public/jsx/views/card-listing.jsx

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+var $ = require('jquery');
+var React = require('react');
+var Navigation = require('react-router').Navigation;
+
+var CardList = require('components/card-list');
+var Spinner = require('components/spinner');
+var gettext = require('utils').gettext;
+
+
+module.exports = React.createClass({
+
+  displayName: 'CardListingView',
+
+  propTypes: {
+    apiSource: React.PropTypes.string.isRequired,
+  },
+
+  mixins: [Navigation],
+
+  getDefaultProps: function() {
+    return {
+      apiSource: '/api/braintree/mozilla/paymethod/',
```

Comment:
I guess not, no - I'll rip those out. They probably should all get move to a settings files at some point.